deploy/scripts/src/spot/requestGenerator/spotReqGen.ts

```diff
@@ -6,9 +6,9 @@ export function generateRequest (currTime: string): SpotRequest {
-        'eyJhbGciOiJFUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJ1cm46ZmRjOmdvdi51azoyMDIyOnMzQkkwQjh3NHNvUEdnQm5aX3ktVXVyMWJKNWFYRUN2aUNram1XZThNR2ciLCJuYmYiOjE2NTIwMjAxMTAsImlzcyI6InBhc3Nwb3J0Y3JpIiwidm90IjoiUDIiLCJleHAiOjE2ODM2MjgxMTAsImlhdCI6MTY1MjA5MjExMCwidmMiOnsiZXZpZGVuY2UiOlt7ImFjdGl2aXR5SGlzdG9yeVNjb3JlIjowLCJ2YWxpZGl0eVNjb3JlIjoyLCJ2ZXJpZmljYXRpb25TY29yZSI6MCwic3RyZW5ndGhTY29yZSI6NCwidHhuIjoidHhuIiwiaWRlbnRpdHlGcmF1ZFNjb3JlIjowLCJ0eXBlIjoiSWRlbnRpdHlDaGVjayJ9XSwiY3JlZGVudGlhbFN1YmplY3QiOnsibmFtZSI6W3sibmFtZVBhcnRzIjpbeyJ0eXBlIjoiR2l2ZW5OYW1lIiwidmFsdWUiOiJKYW5lIn0seyJ0eXBlIjoiRmFtaWx5TmFtZSIsInZhbHVlIjoiV3JpZ2h0In1dLCJ2YWxpZEZyb20iOiIyMDE5LTA0LTAxIn0seyJ2YWxpZFVudGlsIjoiMjAyOS0wNC0wMSIsIm5hbWVQYXJ0cyI6W3sidHlwZSI6IkdpdmVuTmFtZSIsInZhbHVlIjoiSmFuZSJ9LHsidHlwZSI6IkZhbWlseU5hbWUiLCJ2YWx1ZSI6IldyaWdodCJ9XX1dLCJiaXJ0aERhdGUiOlt7InZhbHVlIjoiMTk4OS0wNy0wNiJ9XX0sInR5cGUiOlsiVmVyaWZpYWJsZUNyZWRlbnRpYWwiLCJJZGVudGl0eUNoZWNrQ3JlZGVudGlhbCJdLCJAY29udGV4dCI6WyJodHRwczovL3d3dy53My5vcmcvMjAxOC9jcmVkZW50aWFscy92MSIsImh0dHBzOi8vdm9jYWIubG9uZG9uLmNsb3VkYXBwcy5kaWdpdGFsL2NvbnRleHRzL2lkZW50aXR5LXYxLmpzb25sZCJdfSwidnRtIjoiaHR0cHM6Ly9vaWRjLmFjY291bnQuZ292LnVrL3RydXN0bWFyayJ9.MEQCIARWju22taR5B-kR6pnJnqW9NHNBotETORbHskk4MUkkAiAEWNtEr8dK25gZaxKjSyJJ-1Eikt1qSCzldHuSO5Xuyg' // pragma: allowlist secret
+        'eyJhbGciOiJFUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJ1cm46ZmRjOmdvdi51azoyMDIyOkNZN1E2TDd4MEFlQWNmYUdlazY2R2FKeVhhM2R4RVhBaDRxWWtETUVzMUVzZWN0IiwibmJmIjoxNjkxMzE3MjkwLCJpc3MiOiJmcmF1ZGNyaSIsInZvdCI6IlAyIiwiZXhwIjoxNjk0MDIwODkwLCJpYXQiOjE2OTEzNDI0OTAsInZjIjp7ImV2aWRlbmNlIjpbeyJhY3Rpdml0eUhpc3RvcnlTY29yZSI6MCwidmFsaWRpdHlTY29yZSI6MCwidmVyaWZpY2F0aW9uU2NvcmUiOjAsInN0cmVuZ3RoU2NvcmUiOjAsInR4biI6InR4biIsImlkZW50aXR5RnJhdWRTY29yZSI6MSwidHlwZSI6IklkZW50aXR5Q2hlY2sifV0sImNyZWRlbnRpYWxTdWJqZWN0Ijp7Im5hbWUiOlt7Im5hbWVQYXJ0cyI6W3sidHlwZSI6IkdpdmVuTmFtZSIsInZhbHVlIjoiSmFuZSJ9LHsidHlwZSI6IkZhbWlseU5hbWUiLCJ2YWx1ZSI6IldyaWdodCJ9XSwidmFsaWRGcm9tIjoiMjAxOS0wNC0wMSJ9LHsidmFsaWRVbnRpbCI6IjIwMjktMDQtMDEiLCJuYW1lUGFydHMiOlt7InR5cGUiOiJHaXZlbk5hbWUiLCJ2YWx1ZSI6IkphbmUifSx7InR5cGUiOiJGYW1pbHlOYW1lIiwidmFsdWUiOiJXcmlnaHQifV19XSwiYmlydGhEYXRlIjpbeyJ2YWx1ZSI6IjE5ODktMDctMDYifV19LCJ0eXBlIjpbIlZlcmlmaWFibGVDcmVkZW50aWFsIiwiSWRlbnRpdHlDaGVja0NyZWRlbnRpYWwiXSwiQGNvbnRleHQiOlsiaHR0cHM6Ly93d3cudzMub3JnLzIwMTgvY3JlZGVudGlhbHMvdjEiLCJodHRwczovL3ZvY2FiLmxvbmRvbi5jbG91ZGFwcHMuZGlnaXRhbC9jb250ZXh0cy9pZGVudGl0eS12MS5qc29ubGQiXX0sInZ0bSI6Imh0dHBzOi8vb2lkYy5hY2NvdW50Lmdvdi51ay90cnVzdG1hcmsifQ==.MEUCIQDvawAy4b1lX1NpHS3yO7DL360ZJ4baPcmYXgaG6__0eAIgEsNylyn45DIXTI71k0NvnNqScwQxjWF-TYmaAdW_F4A', // pragma: allowlist secret
```

Comment:
These JWTs are not valid. It shouldn't have any padding `=` characters in the payload and the signature will need to be updated as the payload has changed

Reply:
Updated the JWT tokens in the next commit. These tokens have been generated using the `create-jwt` utility in the `di-ipv-spot-dev-tools` repository